### PR TITLE
Remove Current User From List of Default Reviewer

### PR DESCRIPTION
**Pull request by Kimor-hello**
Creating a pull request on bitbucket automatically fetch all default reviewer and add it to the reviewer list. However if the user logged in to use the dependabot script on bitbucket is one of the default reviewer, the pull request creation will fail.
To avoid this, this fetch the current user uuid and remove it from the default reviewer list if necessary.

### CODE REVIEW

File: common/.ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+3.1.2
```
Comment (jeffwidman) [inline common/.ruby-version:1]: Is there a reason for this? We already have https://github.com/dependabot/dependabot-core/blob/efaca437c99258fdedf61ec7d6e498346aa59f44/.ruby-version

**Review by jeffwidman — COMMENTED** [review]
Looks fine to me other than I am unclear why add the `.ruby-version` file?

To be clear, I have no way to integration test the client, it's very much a "we're happy to sanity check the code, but otherwise up to the community to maintain". Thank you for the unit test, that certainly helps.

File: common/.ruby-version (hunk 1 above)
Comment (Kimor-hello) [reply common/.ruby-version:1]: Hello, 

Sorry no, I was testing some options in you read me in order to test it in my own custom usage with dependabot script, will remove it.

**Review by jeffwidman — APPROVED** [review]
Looks good, thanks for including the unit test!

**Pull request merged**